### PR TITLE
Added InfoToggle component

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-639

#### Description
Added a component to support the info tooltip pattern we've been seeing a lot of lately. This will replace [InfoTooltip in mup-web](https://github.com/meetup/mup-web/blob/c3219b603abd77e7635baf521a260da793553499/src/components/InfoTooltip.jsx)

#### Screenshots (if applicable)
screen shot 2018-05-24 at 2 53 37 pm

### CODE REVIEW

File: src/interactive/__snapshots__/infoToggle.test.jsx.snap [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`InfoToggle should render 1`] = `
+<div>
+  <span />
```
Comment (akdetrick) [inline src/interactive/__snapshots__/infoToggle.test.jsx.snap]: Looks like a label prop needs to be mocked

File: src/utils/components/FloatingPosition.jsx [hunk 2]
```diff
@@ -128,6 +129,8 @@ class FloatingPosition extends React.PureComponent {
 	}
 
 	render() {
+		const throttledMaxWidth = this.props.media.isAtSmallUp ? '350px' : '200px';
```
Comment (akdetrick) [inline src/utils/components/FloatingPosition.jsx]: "throttle" implies handling something like a stream of events... `boundedMaxWidth` may be more descriptive of what this render prop provides.

Comment (akdetrick) [reply src/utils/components/FloatingPosition.jsx]: just `maxWidth` would also work here; it's probably not necessary to expose _how_ at works as much as what the prop value should be used for.

Comment (mperrotti) [reply src/utils/components/FloatingPosition.jsx]: I'm going to use `boundedMaxWidth` so I can still do `maxWidth={maxWidth || boundedMaxWidth}` in `Dropdown` and `Toggle` and let people override what gets passed through `FloatingPosition`

**Comment by mperrotti** [comment]
@amandakievet - fixed it in this PR: https://github.com/meetup/meetup-web-components/pull/549

**Pull request merged**